Execution crashes on startup

Project (reduced from a big setup):
```
from Standard.Base import all
from Standard.Table import all
from Standard.Database import all
from Standard.AWS import all
from Standard.Geo import all
from Standard.Google_Api import all
from Standard.Microsoft import all
from Standard.Snowflake import all
from Standard.Tableau import all
import Standard.Examples
import Standard.Visualization

polyglot java import java.lang.Thread

main =
    node2 = Standard.Base.Enso_Cloud.Internal.Utils
    node3 = node2.get_cached "foo" (Thread.sleep 10000)
    node4 = node2.invalidate_cache "foo"
    
    file1 = Data.read Examples.welcome
```
can lead to
```
[ERROR] [2025-04-25T14:21:24.014] [org.enso.interpreter.instrument.job.ExecuteJob] Failed to execute
java.lang.ExceptionInInitializerError: null
	at java.base@24.0.1/java.lang.Class.ensureInitialized(DynamicHub.java:658)
	at java.base@24.0.1/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:136)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.reflect.fieldaccessor.UnsafeFieldAccessorFactory.newFieldAccessor(UnsafeFieldAccessorFactory.java:44)
	at java.base@24.0.1/jdk.internal.reflect.ReflectionFactory.newFieldAccessor(ReflectionFactory.java:2165)
	at java.base@24.0.1/java.lang.reflect.Field.acquireFieldAccessor(Field.java:1180)
	at java.base@24.0.1/java.lang.reflect.Field.getFieldAccessor(Field.java:107)
	at java.base@24.0.1/java.lang.reflect.Field.get(Field.java:438)
	at org.graalvm.truffle/com.oracle.truffle.host.HostFieldDesc$ReflectImpl.reflectGet(HostFieldDesc.java:134)
	at org.graalvm.truffle/com.oracle.truffle.host.HostFieldDesc$ReflectImpl.get(HostFieldDesc.java:117)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObject$ReadFieldNode.doCached(HostObject.java:3776)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectFactory$ReadFieldNodeGen$Inlined.executeAndSpecialize(HostObjectFactory.java:1471)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectFactory$ReadFieldNodeGen$Inlined.execute(HostObjectFactory.java:1441)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObject.readMember(HostObject.java:317)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectGen$InteropLibraryExports$Cached.readMemberNode_AndSpecialize(HostObjectGen.java:6924)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectGen$InteropLibraryExports$Cached.readMember(HostObjectGen.java:6914)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.readMember(InteropLibraryGen.java:8376)
	at org.enso.interpreter.node.callable.resolver.HostMethodCallNode.resolveHostField(HostMethodCallNode.java:258)
	at org.enso.interpreter.node.callable.resolver.HostMethodCallNodeGen.executeAndSpecialize(HostMethodCallNodeGen.java:189)
	at org.enso.interpreter.node.callable.resolver.HostMethodCallNodeGen.execute(HostMethodCallNodeGen.java:114)
	at org.enso.interpreter.node.callable.InvokeMethodNode.doPolyglot(InvokeMethodNode.java:562)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.executeAndSpecialize(InvokeMethodNodeGen.java:868)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.execute(InvokeMethodNodeGen.java:525)
	at org.enso.interpreter.node.callable.InvokeCallableNode.invokeDynamicSymbol(InvokeCallableNode.java:286)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.executeAndSpecialize(InvokeCallableNodeGen.java:274)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.execute(InvokeCallableNodeGen.java:226)
	at org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:97)
	at org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:94)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNode.doCached(ThunkExecutorNode.java:74)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNodeGen.executeAndSpecialize(ThunkExecutorNodeGen.java:203)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNodeGen.executeThunk(ThunkExecutorNodeGen.java:163)
	at org.enso.interpreter.node.callable.InvokeCallableNode.invokeDynamicSymbol(InvokeCallableNode.java:283)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.executeAndSpecialize(InvokeCallableNodeGen.java:274)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.execute(InvokeCallableNodeGen.java:226)
	at org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:97)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNode.callDirect(ExecuteCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeAndSpecialize(ExecuteCallNodeGen.java:168)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeCall(ExecuteCallNodeGen.java:98)
	at org.enso.interpreter.node.callable.dispatch.SimpleCallOptimiserNode.executeDispatch(SimpleCallOptimiserNode.java:56)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.doCall(CurryNode.java:162)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.execute(CurryNode.java:108)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNode.invokeCached(InvokeFunctionNode.java:157)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.executeAndSpecialize(InvokeFunctionNodeGen.java:132)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.execute(InvokeFunctionNodeGen.java:94)
	at org.enso.interpreter.node.callable.InvokeMethodNode.doFunctionalDispatchCachedSymbol(InvokeMethodNode.java:161)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.executeAndSpecialize(InvokeMethodNodeGen.java:604)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.execute(InvokeMethodNodeGen.java:525)
	at org.enso.interpreter.node.callable.InvokeCallableNode.invokeDynamicSymbol(InvokeCallableNode.java:286)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.executeAndSpecialize(InvokeCallableNodeGen.java:274)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.execute(InvokeCallableNodeGen.java:226)
	at org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:97)
	at org.enso.interpreter.node.ExpressionNodeWrapper.executeGeneric(ExpressionNodeWrapper.java:37)
	at org.enso.interpreter.node.scope.AssignmentNodeGen.executeGeneric(AssignmentNodeGen.java:48)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:70)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNode.callDirect(ExecuteCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeAndSpecialize(ExecuteCallNodeGen.java:168)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeCall(ExecuteCallNodeGen.java:98)
	at org.enso.interpreter.node.callable.dispatch.SimpleCallOptimiserNode.executeDispatch(SimpleCallOptimiserNode.java:56)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.doCall(CurryNode.java:162)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.execute(CurryNode.java:108)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNode.invokeCached(InvokeFunctionNode.java:157)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.executeAndSpecialize(InvokeFunctionNodeGen.java:132)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.execute(InvokeFunctionNodeGen.java:94)
	at org.enso.interpreter.node.callable.InteropApplicationNode.callCached(InteropApplicationNode.java:84)
	at org.enso.interpreter.node.callable.InteropApplicationNodeGen.executeAndSpecialize(InteropApplicationNodeGen.java:144)
	at org.enso.interpreter.node.callable.InteropApplicationNodeGen.execute(InteropApplicationNodeGen.java:95)
	at org.enso.interpreter.node.callable.FunctionCallInstrumentationNode$FunctionCall$Execute.callCached(FunctionCallInstrumentationNode.java:104)
	at org.enso.interpreter.node.callable.FunctionCallGen$InteropLibraryExports$Cached.executeAndSpecialize(FunctionCallGen.java:101)
	at org.enso.interpreter.node.callable.FunctionCallGen$InteropLibraryExports$Cached.execute(FunctionCallGen.java:90)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.execute(InteropLibraryGen.java:7900)
	at org.enso.interpreter.service.ExecutionService$ExecuteRootNode.execute(ExecutionService.java:619)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.call(OptimizedCallTarget.java:545)
	at org.graalvm.truffle/com.oracle.truffle.api.nodes.IndirectCallNode$1.call(IndirectCallNode.java:96)
	at org.enso.interpreter.node.callable.ExecuteCallNode.callIndirect(ExecuteCallNode.java:128)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen$Uncached.executeCall(ExecuteCallNodeGen.java:242)
	at org.enso.interpreter.node.callable.dispatch.LoopingCallOptimiserNode.loopUntilCompletion(LoopingCallOptimiserNode.java:137)
	at org.enso.interpreter.node.callable.dispatch.LoopingCallOptimiserNode.uncachedDispatch(LoopingCallOptimiserNode.java:109)
	at org.enso.interpreter.node.callable.dispatch.LoopingCallOptimiserNodeGen$Uncached.executeDispatch(LoopingCallOptimiserNodeGen.java:267)
	at org.enso.interpreter.node.callable.dispatch.IndirectCurryNode.doCall(IndirectCurryNode.java:134)
	at org.enso.interpreter.node.callable.dispatch.IndirectCurryNode.doCurry(IndirectCurryNode.java:78)
	at org.enso.interpreter.node.callable.dispatch.IndirectCurryNodeGen$Uncached.execute(IndirectCurryNodeGen.java:119)
	at org.enso.interpreter.node.callable.dispatch.IndirectInvokeFunctionNode.invokeUncached(IndirectInvokeFunctionNode.java:83)
	at org.enso.interpreter.node.callable.dispatch.IndirectInvokeFunctionNodeGen$Uncached.execute(IndirectInvokeFunctionNodeGen.java:118)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNode.doUncachedFn(ThunkExecutorNode.java:138)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNodeGen$Uncached.executeThunk(ThunkExecutorNodeGen.java:378)
	at org.enso.interpreter.runtime.state.RunStateNode.doFresh(RunStateNode.java:89)
	at org.enso.interpreter.runtime.state.RunStateNodeGen$Uncached.execute(RunStateNodeGen.java:269)
	at org.enso.interpreter.service.ExecutionService.lambda$execute$1(ExecutionService.java:214)
	at org.enso.interpreter.runtime.ThreadManager.lambda$submit$2(ThreadManager.java:76)
	at java.base@24.0.1/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1814)
	at java.base@24.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base@24.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base@24.0.1/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:309)
	at java.base@24.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base@24.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadTask$ThreadSpawnRootNode.executeImpl(PolyglotThreadTask.java:144)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadTask$ThreadSpawnRootNode.execute(PolyglotThreadTask.java:135)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.call(OptimizedCallTarget.java:562)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadTask.run(PolyglotThreadTask.java:118)
	at java.base@24.0.1/java.lang.Thread.runWith(Thread.java:1460)
	at java.base@24.0.1/java.lang.Thread.run(Thread.java:1447)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:832)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:808)
	Suppressed: com.oracle.truffle.api.TruffleStackTrace$LazyStackTrace: null
Caused by: org.enso.interpreter.runtime.control.ThreadInterruptedException: null
	at org.enso.interpreter.runtime.ThreadManager$1.perform(ThreadManager.java:138)
	at org.graalvm.truffle/com.oracle.truffle.api.LanguageAccessor$LanguageImpl.performTLAction(LanguageAccessor.java:566)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadLocalActions$AsyncEvent.acceptImpl(PolyglotThreadLocalActions.java:783)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadLocalActions$AbstractTLHandshake.accept(PolyglotThreadLocalActions.java:730)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadLocalActions$AbstractTLHandshake.accept(PolyglotThreadLocalActions.java:627)
	at org.graalvm.truffle/com.oracle.truffle.api.impl.ThreadLocalHandshake$Handshake.perform(ThreadLocalHandshake.java:355)
	at org.graalvm.truffle/com.oracle.truffle.api.impl.ThreadLocalHandshake$TruffleSafepointImpl.processOrNotifyHandshakes(ThreadLocalHandshake.java:633)
	at org.graalvm.truffle/com.oracle.truffle.api.impl.ThreadLocalHandshake.processHandshake(ThreadLocalHandshake.java:174)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateThreadLocalHandshake.invokeProcessHandshake(SubstrateThreadLocalHandshake.java:127)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateThreadLocalHandshake.poll(SubstrateThreadLocalHandshake.java:86)
	at org.graalvm.truffle/com.oracle.truffle.api.TruffleSafepoint.poll(TruffleSafepoint.java:151)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:820)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedRuntimeSupport.callProfiled(OptimizedRuntimeSupport.java:291)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotValueDispatch$InteropValue.invoke(PolyglotValueDispatch.java:2687)
	at org.graalvm.polyglot/org.graalvm.polyglot.Value.invokeMember(Value.java:1157)
	at org.enso.base.polyglot.EnsoMeta.callStaticModuleMethod(EnsoMeta.java:29)
	at org.enso.base.cache.ReloadDetector$ReloadSentinel.resetEnsoReloadSentinel(ReloadDetector.java:85)
	at org.enso.base.cache.ReloadDetector$ReloadSentinel.<init>(ReloadDetector.java:72)
	at org.enso.base.cache.ReloadDetector.register(ReloadDetector.java:29)
	at org.enso.base.enso_cloud.CloudRequestCache.<init>(CloudRequestCache.java:11)
	at org.enso.base.enso_cloud.CloudRequestCache.<clinit>(CloudRequestCache.java:14)
	... 147 common frames omitted
	Suppressed: com.oracle.truffle.api.TruffleStackTrace$LazyStackTrace: null
[ERROR] [2025-04-25T14:21:24.769] [org.enso.interpreter.instrument.job.ExecuteJob] Failed to execute
java.lang.NoClassDefFoundError: Could not initialize class org.enso.base.enso_cloud.CloudRequestCache
	at java.base@24.0.1/java.lang.Class.ensureInitialized(DynamicHub.java:658)
	at java.base@24.0.1/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:136)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.reflect.fieldaccessor.UnsafeFieldAccessorFactory.newFieldAccessor(UnsafeFieldAccessorFactory.java:44)
	at java.base@24.0.1/jdk.internal.reflect.ReflectionFactory.newFieldAccessor(ReflectionFactory.java:2165)
	at java.base@24.0.1/java.lang.reflect.Field.acquireFieldAccessor(Field.java:1180)
	at java.base@24.0.1/java.lang.reflect.Field.getFieldAccessor(Field.java:107)
	at java.base@24.0.1/java.lang.reflect.Field.get(Field.java:438)
	at org.graalvm.truffle/com.oracle.truffle.host.HostFieldDesc$ReflectImpl.reflectGet(HostFieldDesc.java:134)
	at org.graalvm.truffle/com.oracle.truffle.host.HostFieldDesc$ReflectImpl.get(HostFieldDesc.java:117)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObject$ReadFieldNode.doCached(HostObject.java:3776)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectFactory$ReadFieldNodeGen$Inlined.executeAndSpecialize(HostObjectFactory.java:1471)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectFactory$ReadFieldNodeGen$Inlined.execute(HostObjectFactory.java:1441)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObject.readMember(HostObject.java:317)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectGen$InteropLibraryExports$Cached.readMemberNode_AndSpecialize(HostObjectGen.java:6924)
	at org.graalvm.truffle/com.oracle.truffle.host.HostObjectGen$InteropLibraryExports$Cached.readMember(HostObjectGen.java:6914)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.readMember(InteropLibraryGen.java:8376)
	at org.enso.interpreter.node.callable.resolver.HostMethodCallNode.resolveHostField(HostMethodCallNode.java:258)
	at org.enso.interpreter.node.callable.resolver.HostMethodCallNodeGen.executeAndSpecialize(HostMethodCallNodeGen.java:189)
	at org.enso.interpreter.node.callable.resolver.HostMethodCallNodeGen.execute(HostMethodCallNodeGen.java:114)
	at org.enso.interpreter.node.callable.InvokeMethodNode.doPolyglot(InvokeMethodNode.java:562)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.executeAndSpecialize(InvokeMethodNodeGen.java:868)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.execute(InvokeMethodNodeGen.java:525)
	at org.enso.interpreter.node.callable.InvokeCallableNode.invokeDynamicSymbol(InvokeCallableNode.java:286)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.executeAndSpecialize(InvokeCallableNodeGen.java:274)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.execute(InvokeCallableNodeGen.java:226)
	at org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:97)
	at org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:94)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNode.doCached(ThunkExecutorNode.java:74)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNodeGen.executeAndSpecialize(ThunkExecutorNodeGen.java:203)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNodeGen.executeThunk(ThunkExecutorNodeGen.java:163)
	at org.enso.interpreter.node.callable.InvokeCallableNode.invokeDynamicSymbol(InvokeCallableNode.java:283)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.executeAndSpecialize(InvokeCallableNodeGen.java:274)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.execute(InvokeCallableNodeGen.java:226)
	at org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:97)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNode.callDirect(ExecuteCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeAndSpecialize(ExecuteCallNodeGen.java:168)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeCall(ExecuteCallNodeGen.java:98)
	at org.enso.interpreter.node.callable.dispatch.SimpleCallOptimiserNode.executeDispatch(SimpleCallOptimiserNode.java:56)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.doCall(CurryNode.java:162)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.execute(CurryNode.java:108)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNode.invokeCached(InvokeFunctionNode.java:157)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.executeAndSpecialize(InvokeFunctionNodeGen.java:132)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.execute(InvokeFunctionNodeGen.java:94)
	at org.enso.interpreter.node.callable.InvokeMethodNode.doFunctionalDispatchCachedSymbol(InvokeMethodNode.java:161)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.executeAndSpecialize(InvokeMethodNodeGen.java:604)
	at org.enso.interpreter.node.callable.InvokeMethodNodeGen.execute(InvokeMethodNodeGen.java:525)
	at org.enso.interpreter.node.callable.InvokeCallableNode.invokeDynamicSymbol(InvokeCallableNode.java:286)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.executeAndSpecialize(InvokeCallableNodeGen.java:274)
	at org.enso.interpreter.node.callable.InvokeCallableNodeGen.execute(InvokeCallableNodeGen.java:226)
	at org.enso.interpreter.node.callable.ApplicationNode.executeGeneric(ApplicationNode.java:97)
	at org.enso.interpreter.node.ExpressionNodeWrapper.executeGeneric(ExpressionNodeWrapper.java:37)
	at org.enso.interpreter.node.scope.AssignmentNodeGen.executeGeneric(AssignmentNodeGen.java:48)
	at org.enso.interpreter.node.ExpressionNodeWrapper.executeGeneric(ExpressionNodeWrapper.java:37)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:70)
	at org.enso.interpreter.node.ExpressionNodeWrapper.executeGeneric(ExpressionNodeWrapper.java:37)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.callable.function.BlockNode.executeGeneric(BlockNode.java:78)
	at org.enso.interpreter.node.ClosureRootNode.execute(ClosureRootNode.java:85)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedDirectCallNode.call(OptimizedDirectCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNode.callDirect(ExecuteCallNode.java:94)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeAndSpecialize(ExecuteCallNodeGen.java:168)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen.executeCall(ExecuteCallNodeGen.java:98)
	at org.enso.interpreter.node.callable.dispatch.SimpleCallOptimiserNode.executeDispatch(SimpleCallOptimiserNode.java:56)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.doCall(CurryNode.java:162)
	at org.enso.interpreter.node.callable.dispatch.CurryNode.execute(CurryNode.java:108)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNode.invokeCached(InvokeFunctionNode.java:157)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.executeAndSpecialize(InvokeFunctionNodeGen.java:132)
	at org.enso.interpreter.node.callable.dispatch.InvokeFunctionNodeGen.execute(InvokeFunctionNodeGen.java:94)
	at org.enso.interpreter.node.callable.InteropApplicationNode.callCached(InteropApplicationNode.java:84)
	at org.enso.interpreter.node.callable.InteropApplicationNodeGen.execute(InteropApplicationNodeGen.java:75)
	at org.enso.interpreter.node.callable.FunctionCallInstrumentationNode$FunctionCall$Execute.callCached(FunctionCallInstrumentationNode.java:104)
	at org.enso.interpreter.node.callable.FunctionCallGen$InteropLibraryExports$Cached.execute(FunctionCallGen.java:85)
	at org.graalvm.truffle/com.oracle.truffle.api.interop.InteropLibraryGen$CachedDispatch.execute(InteropLibraryGen.java:7900)
	at org.enso.interpreter.service.ExecutionService$ExecuteRootNode.execute(ExecutionService.java:619)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.call(OptimizedCallTarget.java:545)
	at org.graalvm.truffle/com.oracle.truffle.api.nodes.IndirectCallNode$1.call(IndirectCallNode.java:96)
	at org.enso.interpreter.node.callable.ExecuteCallNode.callIndirect(ExecuteCallNode.java:128)
	at org.enso.interpreter.node.callable.ExecuteCallNodeGen$Uncached.executeCall(ExecuteCallNodeGen.java:242)
	at org.enso.interpreter.node.callable.dispatch.LoopingCallOptimiserNode.loopUntilCompletion(LoopingCallOptimiserNode.java:137)
	at org.enso.interpreter.node.callable.dispatch.LoopingCallOptimiserNode.uncachedDispatch(LoopingCallOptimiserNode.java:109)
	at org.enso.interpreter.node.callable.dispatch.LoopingCallOptimiserNodeGen$Uncached.executeDispatch(LoopingCallOptimiserNodeGen.java:267)
	at org.enso.interpreter.node.callable.dispatch.IndirectCurryNode.doCall(IndirectCurryNode.java:134)
	at org.enso.interpreter.node.callable.dispatch.IndirectCurryNode.doCurry(IndirectCurryNode.java:78)
	at org.enso.interpreter.node.callable.dispatch.IndirectCurryNodeGen$Uncached.execute(IndirectCurryNodeGen.java:119)
	at org.enso.interpreter.node.callable.dispatch.IndirectInvokeFunctionNode.invokeUncached(IndirectInvokeFunctionNode.java:83)
	at org.enso.interpreter.node.callable.dispatch.IndirectInvokeFunctionNodeGen$Uncached.execute(IndirectInvokeFunctionNodeGen.java:118)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNode.doUncachedFn(ThunkExecutorNode.java:138)
	at org.enso.interpreter.node.callable.thunk.ThunkExecutorNodeGen$Uncached.executeThunk(ThunkExecutorNodeGen.java:378)
	at org.enso.interpreter.runtime.state.RunStateNode.doFresh(RunStateNode.java:89)
	at org.enso.interpreter.runtime.state.RunStateNodeGen$Uncached.execute(RunStateNodeGen.java:269)
	at org.enso.interpreter.service.ExecutionService.lambda$execute$1(ExecutionService.java:214)
	at org.enso.interpreter.runtime.ThreadManager.lambda$submit$2(ThreadManager.java:76)
	at java.base@24.0.1/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1814)
	at java.base@24.0.1/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:545)
	at java.base@24.0.1/java.util.concurrent.FutureTask.run(FutureTask.java:328)
	at java.base@24.0.1/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:309)
	at java.base@24.0.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1095)
	at java.base@24.0.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:619)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadTask$ThreadSpawnRootNode.executeImpl(PolyglotThreadTask.java:144)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadTask$ThreadSpawnRootNode.execute(PolyglotThreadTask.java:135)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.executeRootNode(OptimizedCallTarget.java:819)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.profiledPERoot(OptimizedCallTarget.java:743)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callBoundary(OptimizedCallTarget.java:667)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.invokeCallBoundary(SubstrateOptimizedCallTarget.java:124)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTargetInstalledCode.doInvoke(SubstrateOptimizedCallTargetInstalledCode.java:232)
	at org.graalvm.truffle.runtime.svm/com.oracle.svm.truffle.api.SubstrateOptimizedCallTarget.doInvoke(SubstrateOptimizedCallTarget.java:106)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.callDirect(OptimizedCallTarget.java:599)
	at org.graalvm.truffle.runtime/com.oracle.truffle.runtime.OptimizedCallTarget.call(OptimizedCallTarget.java:562)
	at org.graalvm.truffle/com.oracle.truffle.polyglot.PolyglotThreadTask.run(PolyglotThreadTask.java:118)
	at java.base@24.0.1/java.lang.Thread.runWith(Thread.java:1460)
	at java.base@24.0.1/java.lang.Thread.run(Thread.java:1447)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:832)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.thread.PlatformThreads.threadStartRoutine(PlatformThreads.java:808)
	Suppressed: com.oracle.truffle.api.TruffleStackTrace$LazyStackTrace: null
Language Server transport error: ErrorEvent {

```

Something about how `ReloadSentinel` and `org.enso.base.enso_cloud.CloudRequestCache` are defined, which makes it susceptible to a crash.

--- a/std-bits/base/src/main/java/org/enso/base/cache/ReloadDetector.java
+++ b/std-bits/base/src/main/java/org/enso/base/cache/ReloadDetector.java
@@ -67,12 +67,21 @@ public class ReloadDetector {
    */
   private static class ReloadSentinel {
     private Value ensoReloadSentinel;
+    private boolean initialized;
 
     public ReloadSentinel() {
-      resetEnsoReloadSentinel();
+      initialized = false;
+    }
+
+    private void ensureInitialized() {
+      if (!initialized) {
+        resetEnsoReloadSentinel();
+      }
+      initialized = true;
     }
 
     public boolean hasReloadOccurred() {
+      ensureInitialized();
       var reloadHasOccurred = ensoReloadSentinel.invokeMember("has_reload_occurred").asBoolean();
       if (reloadHasOccurred) {
         resetEnsoReloadSentinel();
@@ -87,6 +96,7 @@ public class ReloadDetector {
     }
 
     public void simulateReloadTestOnly() {
+      ensureInitialized();
       EnsoMeta.callStaticModuleMethod(
           "Standard.Base.Network.Reload_Sentinel", "simulate_reload_test_only", ensoReloadSentinel);
     }

--- a/std-bits/base/src/main/resources/META-INF/native-image/org/enso/base/reflect-config.json
+++ b/std-bits/base/src/main/resources/META-INF/native-image/org/enso/base/reflect-config.json
@@ -160,6 +160,21 @@
         "name": "org.enso.base.enso_cloud.audit.AuditLog"
     },
     {
+        "name": "org.enso.base.enso_cloud.audit.AuditLog$AuditLogError"
+    },
+    {
+        "name": "org.enso.base.enso_cloud.audit.AuditLog.AuditLogError"
+    },
+    {
+        "name": "org.enso.base.enso_cloud.audit.AuditLogApiAccess",
+        "fields": [
+            {
+                "name": "INSTANCE"
+            }
+        ],
+        "allPublicMethods": true
+    },
+    {
         "name": "org.enso.base.file_format.FileFormatSPI"
     },
     {
@@ -491,6 +506,15 @@
             {
                 "name": "init",
                 "parameterTypes": ["javax.net.ssl.KeyManager[]", "javax.net.ssl.TrustManager[]", "java.security.SecureRandom"]
+            }
+        ]
+    },
+    {
+        "name": "java.io.Console",
+        "methods": [
+            {
+                "name": "isTerminal",
+                "parameterTypes": []
             }
         ]
     }

--- a/test/Base_Tests/src/Native_Image_Imports.enso
+++ b/test/Base_Tests/src/Native_Image_Imports.enso
@@ -2,5 +2,3 @@
 # `env ENSO_LAUNCHER=test sbt buildEngineDistribution`.
 # This module is not actually used - it is scanned by `EnsoLibraryFeature` when NI is built.
 private
-
-polyglot java import java.io.Console as Java_Console

